Snowflake.Data/Configuration/EasyLoggingConfigParser.cs

```diff
@@ -61,5 +66,22 @@ private void Validate(ClientConfig config)
                 EasyLoggingLogLevelExtensions.From(config.CommonProps.LogLevel);
             }
         }
+
+        private void CheckForUnknownFields(string fileContent)
+        {
+            // Parse the specified config file and get the key-value pairs from the "common" section
+            List<string> knownProperties = new List<string>();
+            foreach (var property in typeof(ClientConfigCommonProps).GetProperties())
+            {
+                var jsonPropertyAttribute = property.GetCustomAttribute<JsonPropertyAttribute>();
+                knownProperties.Add(jsonPropertyAttribute.PropertyName);
+            }
+
+            JObject.Parse(fileContent).GetValue("common")
+                .Cast<JProperty>()
+                .Where(property => knownProperties.Contains(property.Name))
```

Comment:
Nice find, yes it was logging the known properties instead because it was missing a negation. And I'll add those different cases for known properties in the test

Reply:
Fixed